### DISCUSSION TITLE
Target temperature changes on its own

### DISCUSSION THREAD

**Discussion by mbdawson** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#237] [General]
I first noticed my target temperature changed on its own months ago when I had the spa exposed to the internet. I worried perhaps someone randomly messed with my settings and locked things up on the firewall. I've noticed on multiple occasions since then I've set my target temperature to 96 and then I'll come back a few days later and it will be 99. This has happened a few times now, and I'm positive I didn't set it to 99. Any ideas what might be causing that? Here's my spa config:
<img width="434" alt="Screen Shot 2022-03-25 at 12 33 56 PM" src="https://user-images.githubusercontent.com/5607598/160173490-dd74fadf-3515-4673-84c3-c0f949bd7087.png">

**Comment by visualapproach** [comment]
Hi! Funny you mention this now! I just noticed my spa increased from 20 to 21 at startup. Didn't pay attention to it then but I see you restart the esp on a daily schedule. I bet that's what's causing it. I may be able to fix it soon but meanwhile you can disable the restarts. May I ask why you do that? I've kept my esp running for months without trouble.

**Reply by visualapproach** [reply]
@mbdawson your problem should be solved along with a homeassistant bugfix. Update to firmware version 2022-03-26 in development branch.
